### PR TITLE
Fixed inertial values 

### PR DESCRIPTION
**Pull request by TextZip**
Addressing #2 
1. Replaced incorrect diaginertia values with correct fullinertia values from source urdf.

### CODE REVIEW

**Comment by rohanpsingh** [comment]
@TextZip Thanks for the PR!
I think there is a problem with the tabs/spaces due to which the diff is not generated properly.  
Could you please fix that? Maybe just don't touch the lines you don't have to modify?

**Comment by TextZip** [comment]
@rohanpsingh I used an XML auto-formatter after I finished editing the files as I felt that the spacing and tabs between layers were not uniform in some places.

I can re-upload with the previous spacing but I felt a clean format will make future edits easy, and regarding the diff. view I noticed that there is an option to view the differences without considering the white spaces which might make the comparison possible (attached pic for reference) 

Please let me know if have to revert back to the previous formatting.
  
image

**Comment by rohanpsingh** [comment]
>  regarding the diff. view I noticed that there is an option to view the differences without considering the white spaces which might make the comparison possible (attached pic for reference)

Wow I did not know about this. 
I can see you changed some diaginertias to fullinertias. Is there a strong reason for this? If there isn't let's keep the changes to a minimum in this PR, and solve only the specifc issue it is meant to solve. 

> Please let me know if have to revert back to the previous formatting.

Yes, can we please revert back to the previous formatting? We can create another PR for fixing the formatting ( :bulb: in fact we can make a github action to automate this part in all of our `*_mj_description` repositories!)

**Comment by TextZip** [comment]
The URDF file had full inertia representation so I used that, I thought that would make things consistent. Should I switch back to the diaginertias .?? 

The MuJoCo XML Reference for fullinertia states that "The compiler computes the eigenvalue decomposition of M and sets the frame orientation and diagonal inertia accordingly. "

**Comment by rohanpsingh** [comment]
> Should I switch back to the diaginertias .??

Yes, let's keep diaginertias - which is what we get from the URDF in the conversion process.

**Comment by TextZip** [comment]
Okayy, old formatting and diaginertias then, on it.

File: xml/aliengo.xml [hunk 1]
```diff
@@ -61,17 +61,17 @@
             <site name="imu" size="0.0005 0.0005 0.0005" pos="0 0 0" />
 	    <!-- front right -->
 	    <body name="FR_hip" pos="0.2399 -0.051 0">
-		<inertial pos="-0.022191 -0.015144 -1.5e-05" quat="0.0135493 0.706835 -0.0117635 0.707151" mass="1.993" diaginertia="0.00558695 0.00491009 0.00290132" />
+		<inertial pos="-0.022191 -0.015144 -1.5e-05" quat="0.0135493 0.706835 -0.0117635 0.707151" mass="1.993" diaginertia="0.002903894 0.004907517 0.005586944" />
```
Comment (rohanpsingh) [inline xml/aliengo.xml:64]: Could you please describe briefly how you computed the new `diaginertia` and `quat` values?

**Pull request closed without merging**